pkg/mock/fileUtils_test.go

```diff
@@ -169,7 +169,7 @@ func TestFilesMockFileRemove(t *testing.T) {
 func TestFilesMockGetwd(t *testing.T) {
 	t.Parallel()
 	t.Run("test root", func(t *testing.T) {
-		files := FilesMock{}
+		files := FilesMock{Separator: string(os.PathSeparator)}
```

Comment:
This change should be unnecessary. If it is needed, then there is a place where init() is not called, but should be.

Reply:
Then I guess calling `init()` here instead would be the correct way or?

Reply:
No, this is an internal method. If it needs to be called, it is missing in one of the public methods of FilesMock.

Reply:
Oh yeah sure you are right. Actually it is missing in `Getwd()` method. I fixed it now.